api/swagger-spec/oapi-v1.json

```diff
@@ -17700,6 +17700,10 @@
      "customStrategy": {
       "$ref": "v1.CustomBuildStrategy",
       "description": "CustomStrategy holds the parameters to the Custom build strategy"
+     },
+     "jenkinsPipelineStrategy": {
```

Comment:
> 1) I don't think 'start build' button has any meaning here, @bparees @jimmidyson? I tried to click on that button and the build will stay in 'New' forever ;-)

This should be updated by the Jenkins plugin but not sure it is just yet.

> 3) I'm not sure how are we going to stream the logs from Jenkins (if we are going to do that) to console

To support this we'll need to update the REST API to stream it directly from Jenkins. Not sure that's been decided yet though.

4) I guess visualization of the pipeline will be important here

@jstrachan's done some great work visualizing the pipeline & this needs to be pulled in from the fabric8 console as a plugin I assume.